app/src/androidTest/java/au/com/shiftyjelly/pocketcasts/views/helper/ShowNotesHelperTest.kt

```diff
@@ -36,5 +36,30 @@ class ShowNotesHelperTest {
         actual = ShowNotesHelper.convertTimesToLinks("<li><a href=\"https://www.theverge.com/2021/12/21/22848957/lg-dualup-32-inch-4k-ultra-fine-monitors-announced-specs\">LG’s new 16:18 monitor looks like a multitasking powerhouse</a></li>")
         expected = "<li><a href=\"https://www.theverge.com/2021/12/21/22848957/lg-dualup-32-inch-4k-ultra-fine-monitors-announced-specs\">LG’s new 16:18 monitor looks like a multitasking powerhouse</a></li>"
         assertEquals(actual, expected)
+
+        actual = ShowNotesHelper.convertTimesToLinks("<p>Example <a>10:30</a></p>")
+        expected = "<p>Example <a>10:30</a></p>"
+        assertEquals(expected, actual)
```

Comment:
The other tests have the `assertEquals` with parameters as `(actual, expected)`, but according to the docs, the first parameter should be `expected` and the second one should be `actual`.

<img width="435" alt="image" src="https://user-images.githubusercontent.com/12696894/233808975-f422312e-6d9e-4796-84c4-1481daf242f2.png">

Should I keep it inverted like the others? i.e. `assertEquals(actual, expected)`.

Reply:
What you have is good. For consistency, would you mind adjusting others in this class to conform with the documented guidelines? i.e `assertEquals(actual, expected)`

Reply:
Updated.